### DISCUSSION TITLE
polymod >:(

### DISCUSSION THREAD

**Discussion by SolarPowrr** [KadeArchive/Kade-Engine#602] [Q&A]
i ran "haxelib git polymod https://github.com/larsiusprime/polymod.git" in the powershell and it installed fine. i also installed everything else and have visual studio installed everything else is fine. but when i try to  build the game and run "lime build windows" it tells me that the polymod is missing. whenever i try to reinstall it, it says that its already there and tries to update it, which gets it stuck in a loop and doesnt update it. is there any way to either delete the polymod or build without it or something ???

**Comment by puyoxyz** [comment]
send full console output please (in a gist)

**Reply by SolarPowrr** [reply]
if i try to install polymod it'll say:
C:\Users\myname>haxelib git polymod https://github.com/larsiusprime/polymod.git
You already have polymod version git installed.
Updating polymod version git ...
(and then it gets stuck)

and if i try to build it says:
C:\Users\myname>lime build windows
Error: You must have a "project.xml" file or specify another valid project file when using the 'build' command
(this isnt the issue)

and if i change the cd to kade-engine it says:
C:\Users\myname\Kade-Engine>lime build windows
Error: Could not find haxelib "polymod", does it need to be installed?
